manifests/init.pp

```diff
@@ -573,6 +583,8 @@
   $tcp_backlog                   = $::redis::params::tcp_backlog,
   $tcp_keepalive                 = $::redis::params::tcp_keepalive,
   $timeout                       = $::redis::params::timeout,
+  $unixsocket                    = $::redis::unixsocket,
```

Comment:
It seems you forget the `::params` prefix on the both parameters default values. 

@gcelestine can you fix this ?